### PR TITLE
Add harvester-network-controller sub chart

### PR DESCRIPTION
**Pull request by guangbochen**
**Description:**
add harvester-network-controller sub chart to support VLAN network management of the harvester.

Related Issue:
https://github.com/rancher/harvester/issues/201

### CODE REVIEW

File: deploy/charts/harvester/Chart.yaml [hunk 1]
```diff
@@ -46,3 +46,7 @@ dependencies:
     version: 5.0.32
     repository: file://charts/minio
     condition: minio.enabled
+  - name: harvester-network-controller
+    version: 0.1.0
```
Comment (gitlawr) [inline deploy/charts/harvester/Chart.yaml:50]: This is inconsistent with the versions in `harvester-network-controller/Chart.yaml`

File: deploy/charts/harvester/charts/harvester-network-controller/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Harvester-Network-Controller Helm Chart
+
+[Harvester Network Contrller](https://github.com/rancher/harvester-network-controller) is a network controller helps to manage the host network configuration of the Harvester cluster.
```
Comment (gitlawr) [inline deploy/charts/harvester/charts/harvester-network-controller/README.md]: that helps

**Comment by guangbochen** [comment]
@gitlawr updated, thanks.

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**